packages/conversation/src/models/v1/channel-specific-contact-message/channel-specific-contact-message.ts

```diff
@@ -0,0 +1,21 @@
+import { WhatsAppInteractiveNfmReply } from '../whatsapp-interactive-nfm-reply';
+
+/**
+ * A contact message containing a channel specific message (not supported by OMNI types).
+ */
+export interface ChannelSpecificContactMessage {
+  /** The message type. */
+  message_type: 'nfm_reply';
+  /** @see WhatsAppInteractiveNfmReplyMessage */
+  message: WhatsAppInteractiveNfmReplyMessage;
+}
+
+/**
+ * A WhatsApp interactive contact message containing the nfm_reply.
+ */
+interface WhatsAppInteractiveNfmReplyMessage {
+  /** The interactive message type. */
+  type: string;
```

Comment:
Well, if we were to have other kinds of message types, it would make sense to have the `message_type` as a discriminator and then a dedicated structure under `message`. Btw, protobuf is defining a `google.protobuf.Struct` for the `message` property